src/state/lists/hooks.ts

```diff
@@ -66,25 +66,20 @@ function useCombinedTokenMapFromUrls(urls: string[] | undefined): TokenAddressMa
   }, [lists, urls])
 }
 
-// filter out unsupported lists
-export function useActiveListUrls(): string[] | undefined {
-  const activeListUrls = useAppSelector((state) => state.lists.activeListUrls)
-  return useMemo(() => activeListUrls?.filter((url) => !UNSUPPORTED_LIST_URLS.includes(url)), [activeListUrls])
-}
-
 export function useInactiveListUrls(): string[] {
   const lists = useAllLists()
-  const allActiveListUrls = useActiveListUrls()
   return useMemo(
-    () => Object.keys(lists).filter((url) => !allActiveListUrls?.includes(url) && !UNSUPPORTED_LIST_URLS.includes(url)),
-    [lists, allActiveListUrls]
+    () =>
+      Object.keys(lists).filter(
+        (url) => !DEFAULT_ACTIVE_LIST_URLS?.includes(url) && !UNSUPPORTED_LIST_URLS.includes(url)
```

Comment:
you can define this function as a constant outside of here since the lists are constant